src/screens/FeedScreen/RenderList.js

```diff
@@ -171,7 +265,8 @@ const RenderListFeed = (props) => {
             statusVote={voteStatus}
             showScoreButton={showScoreButton}
             onPressScore={() => showScoreAlertDialog(item)}
-            isSelf={item.anonimity ? false : selfUserId === item.actor.id}
+            isSelf={isSelf}
+            isShowDM
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isSelf` prop has been changed and a new `isShowDM` prop has been added. Ensure that these changes don't introduce any unexpected behavior in the Footer component.
```diff
- isSelf={item.anonimity ? false : selfUserId === item.actor.id}

+ isSelf={isSelf}
+ isShowDM
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->